tests/unit/test_pathutil.py

```diff
@@ -16,14 +16,12 @@
 """Tests for "cylc.flow.pathutil"."""
 
 from unittest import TestCase
-from unittest.mock import call, patch, MagicMock
+from unittest.mock import patch, MagicMock
 
 import pytest
 import os
 import logging
 
-from tests.unit.conftest import mock_glbl_cfg
```

Comment:
No need to import from `conftest.py`, pytest does this automatically it seems

Reply:
It does indeed.